app/css/components/tooltips/concept.css

```diff
@@ -91,4 +93,25 @@
             @apply text-darkGray leading-paragraph font-medium;
         }
     }
+
+    &::before {
```

Comment:
I tend to comment these so the next reader understands _why_ this before and after pseudo element is necessary. In this case it's not because of visuals, so it's important!